src/assets/css/App.css

```diff
@@ -4,6 +4,8 @@ body{
 .App {
   text-align: left;
   border-top: 1px solid #D9DBDC;
+  position: relative;
+  min-height: 700px;
```

Comment:
This should be the direct parent for the footer, because the footer position is `absolute`, and I did that to make sure that the footer will be always in the bottom.